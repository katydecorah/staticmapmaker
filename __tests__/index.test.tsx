import { render } from "@testing-library/react";
import HomePage from "@/pages/index";

it("renders homepage unchanged", () => {
  const { container } = render(<HomePage />);
  expect(container).toMatchSnapshot();
});
