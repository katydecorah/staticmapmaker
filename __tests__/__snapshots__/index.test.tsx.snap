// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <header
    class="cover"
  >
    <div
      class="title"
    >
      <h1>
        <a
          href="/"
        >
          <svg
            aria-hidden="true"
            class="header-icon"
            data-icon="map"
            focusable="false"
            height="128"
            role="img"
            viewBox="0 0 128 128"
            width="128"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M5.5 0C2.367 0 0 2.367 0 5.5v117c0 3.133 2.367 5.5 5.5 5.5h117c3.133 0 5.5-2.367 5.5-5.5V5.5c0-3.133-2.367-5.5-5.5-5.5H5.5zM8 8h112v112H8V8zm35.188 2l15 15-9.594 9.594a2 2 0 1 0 2.813 2.813L61 27.813 101.19 68l-50.032 50h5.688l47.156-47.187 14 14v-5.625L106.814 68l11.188-11.188v-5.625l-14 14L63.814 25l15-15H73.19L61 22.186 48.814 10h-5.625zM48 52c-8.8 0-16 7.2-16 16 0 12 16 32 16 32s16-20 16-32c0-8.8-7.2-16-16-16zm0 8c4.4 0 8 3.6 8 8s-3.6 8-8 8-8-3.6-8-8 3.6-8 8-8z"
            />
          </svg>
          <div>
             Static Map Maker
          </div>
        </a>
      </h1>
    </div>
  </header>
  <main
    class="providers"
  >
    <a
      class="provider"
      href="/bing"
    >
      <img
        alt="Bing static map API"
        src="/img/bing.png"
      />
      <div
        class="provider-title"
      >
        Bing
      </div>
    </a>
    <a
      class="provider"
      href="/google"
    >
      <img
        alt="Google static map API"
        src="/img/google.png"
      />
      <div
        class="provider-title"
      >
        Google
      </div>
    </a>
    <a
      class="provider"
      href="/google-street-view"
    >
      <img
        alt="Google Street View static map API"
        src="/img/google-street-view.png"
      />
      <div
        class="provider-title"
      >
        Google Street View
      </div>
    </a>
    <a
      class="provider"
      href="/here"
    >
      <img
        alt="HERE static map API"
        src="/img/here.png"
      />
      <div
        class="provider-title"
      >
        HERE
      </div>
    </a>
    <a
      class="provider"
      href="/mapbox"
    >
      <img
        alt="Mapbox static map API"
        src="/img/mapbox.png"
      />
      <div
        class="provider-title"
      >
        Mapbox
      </div>
    </a>
    <a
      class="provider"
      href="/mapquest"
    >
      <img
        alt="MapQuest static map API"
        src="/img/mapquest.png"
      />
      <div
        class="provider-title"
      >
        MapQuest
      </div>
    </a>
  </main>
</div>
`;
